docs/cmd/graph.md

```diff
@@ -106,6 +111,10 @@ With this flag, it is possible to link to local files (JS, CSS, fonts, etc.) fro
 
 With `--dot` flag provided containerlab will generate the `dot` file instead of serving the topology with embedded HTTP server.
 
+#### mermaid
+
+With `--mermaid` flag provided containerlab will generate the `mermaid` file instead of serving the topology with embedded HTTP server.
+
```

Comment:
Hi @YutaroHayakawa 
Thank you for this sweet contribution, I will review it more thoroughly in the coming days.
One thing I noticed is the new flag `--mermaid-direction`, would it be possible to add a doc entry for this flag explaining which values can it receive?

Reply:
Yep, thanks. Added in the latest push.